src/initial_conditions/initial_conditions.jl

```diff
@@ -1286,6 +1286,63 @@ function gcm_initial_conditions(external_forcing_file, cfsite_number)
     end
 end
 
+struct InterpolatedColumnProfile{I} <: InitialCondition
+    """Initial data condition for a column model. Stored as a tuple of Interpolation objects"""
```

Comment:
```suggestion
"""
    InterpolatedColumnProfile

Initial data condition for a column model. Stored as a tuple of Interpolation objects
"""
struct InterpolatedColumnProfile{I} <: InitialCondition
```
In Julialang, doc strings need to be immediately before the function or type definition 📒